src/libsync/CMakeLists.txt

```diff
@@ -103,6 +111,10 @@ IF (NOT APPLE)
         FILES ${creds_HEADERS}
 	DESTINATION ${INCLUDE_INSTALL_DIR}/${synclib_NAME}/creds
     )
+    INSTALL(
+        FILES ${CMAKE_CURRENT_BINARY_DIR}/owncloudlib.h
+        DESTINATION ${INCLUDE_INSTALL_DIR}/${synclib_NAME}
+    )
```

Comment:
Why is the owncloudlib.h only installed when not on Apple? That will break the FindOwncloudSync.cmake on apple, or not?